specification/compute/resource-manager/Microsoft.Compute/stable/2022-01-03/GalleryRP/gallery.json

```diff
@@ -2098,6 +2098,16 @@
         "manageActions": {
           "$ref": "#/definitions/UserArtifactManage"
         },
+        "settings": {
+          "$ref": "#/definitions/UserArtifactSettings"
+        },
+        "advancedSettings": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Optional. Additional settings to pass to the VMApp extension. For advanced use only."
```

Comment:
Is VMApp the name of the extension? We don't want abbreviations that are not clear.

Reply:
Include the entire Extension name if that is not it.

Reply:
updated it to full name